### ISSUE TITLE
Issue attaching multiple libraries to each node

### ISSUE THREAD

**Issue by Mullefa** [hadley/multidplyr#21]
The documentation infers you can attach multiple packages to each node with a single call to `cluster_library()`. However, this doesn't work, at least in my case:

``` R
get_default_cluster() %>%
  cluster_library(c("dplyr", "geosphere"))

Initialising 7 core cluster.
Error in library(packages, character.only = TRUE) : 
  'package' must be of length 1
```

**Comment by puterleat** [comment]
I think the whole issue of attaching packages is quite confusing at the moment... It took me a while in the docs to realise you basically have to recreate the namespace of functions within each node of the cluster...  I wonder if there's a way to automate this? Could the current environment be copied to each node?

**Comment by hadley** [comment]
Fixed by #43
